### DISCUSSION TITLE
How to setup a Sandia D case

### DISCUSSION THREAD

**Discussion by ghost** [AMReX-Combustion/PeleLMeX#457] [Q&A]
![Screenshot from 2024-12-27 13-21-19](https://github.com/user-attachments/assets/8c288ca1-d346-4432-ad17-f184efd2e240)

Hello,  

I want to setup a Sandia D case, I have performed the analysis in OpenFOAM and I want to compare the results with PeleLMeX. Please if anyone can help me setup the case. I want to start with a 2D case, How can I define a mesh or import an external mesh. 

Thank you

**Comment by grabtheNS** [comment]
In pelelmex, the eb boundary setting does not support the rz type of coordinate system setting, I have a similar problem, if anyone has a better solution let's exchange solutions together!

**Reply by ghost** [reply]
I think there is a way if I can use the physics of PeleLMeX and combine it with OpenFOAM case structure, but it will take a substantial amount of time. For now I am just using the geometry builder inside the PeleLMeX suite.

**Comment by grabtheNS** [comment]
Yes, it seems like a lot of work!
In fact, different burner nozzle configurations affect the flow structure and combustion characteristics at the [article](https://www.sciencedirect.com/science/article/pii/S0010218018300233?via%3Dihub) and it is not now possible to simulate the nozzle structure at the inlet in the rz coordinate system using the eb boundary conditions in pelelmex, which leads to errors in the simulation.

**Reply by ghost** [reply]
Let me try this week. At least I can identity the equations in the PeleLeMeX code and try to create a solver in OpenFOAM. I cannot access the paper, can you download and send it to me.

**Comment by wjge** [comment]
OpenFOAM r-z is not real r-z. It is by using the wedge-shape finite volume mesh. Is that what you did? Instead, you should be able to simulate 1/4 of the flame with Pele on a 3-D mesh. We had a laminar flame that was set up this way, described in Section 6.1 of  https://doi.org/10.1115/1.4064494. You just need to set up the EB for the nozzle if it is actually needed.

**Reply by ghost** [reply]
Yes, the wedge-shape, that is exactly what I used. Please I can you send me the paper here I cannot download it.  I understand, I can use 1/4th of the 3D geometry. I will try and send it to you here.

**Reply by grabtheNS** [reply]
Where can I get the code for this Section 6.1 case, is it inside the Exec folder. I want to try to set up EB
